docs/tools/mbed_cli_2/use.md

```diff
@@ -35,7 +35,7 @@ To create a new Mbed OS project in a specified path:
     - Absolute. `new` will create the folder if it doesn't exist.
     - Relative:
 
-        - If you have already created a project folder, you can use `.`
+        - If you have already created a project folder, you can use `.`, then update file CMakeLists.txt with: `set(APP_TARGET <created a project folder>` )
```

Comment:
I don't believe we want to require users to manually have to modify CMakeLists.txt. Could you instead please raise a bug on [mbed-tools](https://github.com/ARMmbed/mbed-tools/issues)? It may be that we are missing this line in our template.

Reply:
done "mbed-tools new ." doesn't compile #1414

Reply:
Thanks